dataland-frontend/src/frameworks/gdv/BaseFrameworkDefinition.ts

```diff
@@ -0,0 +1,17 @@
+// <--WARNING--> THIS FILE IS AUTO-GENERATED BY THE FRAMEWORK-TOOLBOX AND WILL BE OVERWRITTEN
+import { type BaseFrameworkDefinition } from "@/frameworks/FrameworkDefinition";
+import { type Configuration, type GdvData } from "@clients/backend";
+import { type AxiosInstance } from "axios";
+import { type FrameworkDataApi } from "@/utils/api/UnifiedFrameworkDataApi";
+import { GdvApiClient } from "@/frameworks/gdv/ApiClient";
+
+export class BaseGdvFrameworkDefinition implements BaseFrameworkDefinition<GdvData> {
+  readonly identifier = "gdv";
+  readonly explanation = "Das GDV/VÖB Framework";
```

Comment:
Could be more descriptive. From where is this generated?

Reply:
`GdvFramework.kt` in the framework-toolbox. But it seems like you already found that :)

Reply:
Yes, as i also wrote it in the GdvFramework.kt file, i will close this here